tests/MongoDB.Driver.GridFS.Tests/GridFSFindOptionsTests.cs

```diff
@@ -26,6 +26,26 @@ namespace MongoDB.Driver.GridFS.Tests
 {
     public class GridFSFindOptionsTests
     {
+        [Fact]
+        public void AllowDiskUse_get_should_return_expected_result()
```

Comment:
I think it would be great to add more tests here. For example, that the specified option has been added to the `FindOperation`. But I don't insist. I will be ok with any disition here.

Reply:
I agree, but I decided to limit the scope of the changes to the same kind of tests that were already present.

@jyemin do you see a need to add new tests that go beyond what's already there?

Reply:
No, let's leave it.  Pretty low risk of messing this up IMO.